src/redux/sagas/listitem.saga.js

```diff
@@ -29,7 +29,7 @@ function* getList(action) {
         url: `/api/list/`,
         data: action.payload
 })
-    console.log('back from GET with:', response.data);
+    //console.log('back from GET with:', response.data);
```

Comment:
Can this be removed?

Reply:
100%.  I'll go through all my sagas and remove all extra logs.  They are comforting but now unnecessary